servicetalk-buffer-netty/src/test/java/io/servicetalk/buffer/netty/BufferAllocatorsTest.java

```diff
@@ -34,118 +33,130 @@
 import static io.servicetalk.buffer.netty.BufferAllocators.PREFER_HEAP_ALLOCATOR;
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.greaterThan;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
-@RunWith(Parameterized.class)
 public class BufferAllocatorsTest {
 
-    private final BufferAllocator allocator;
+    private static final String TEST_NAME_FORMAT = "{index}: allocators = {0}";
 
-    public BufferAllocatorsTest(BufferAllocator allocator) {
-        this.allocator = allocator;
-    }
-
-    @Parameterized.Parameters(name = "{index}: allocators = {0}")
-    public static Collection<Object> data() {
+    @SuppressWarnings("unused")
+    private static Collection<Object> data() {
```

Comment:
Consider renaming that to 'allocators' and changing the generic type to `BufferAllocator`